hudi-common/src/main/java/org/apache/hudi/common/table/view/AbstractTableFileSystemView.java

```diff
@@ -73,11 +73,15 @@
 import static org.apache.hudi.common.table.timeline.HoodieTimeline.METADATA_BOOTSTRAP_INSTANT_TS;
 
 /**
- * Common thread-safe implementation for multiple TableFileSystemView Implementations. <br>Provides uniform handling of: <br>(a)
- * Loading file-system views from underlying file-system <br>(b) Pending compaction operations and changing file-system
- * views based on that <br>(c) Thread-safety in loading and managing file system views for this table. <br>(d) resetting
- * file-system views <br>The actual mechanism of fetching file slices from different view storages is delegated to
- * sub-classes.
+ * Common thread-safe implementation for multiple TableFileSystemView Implementations.
+ * Provides uniform handling of:
+ * <ul>
+ *   <li>Loading file-system views from underlying file-system</li>
+ *   <li>Pending compaction operations and changing file-system views based on that</li>
+ *   <li>Thread-safety in loading and managing file system views for this table</li>
```

Comment:
For each of the item, end it with `;`, and for the last item, end with `.`.

Reply:
done, thanks for your guidance